### DISCUSSION TITLE
BLOQUEO AL AJUSTAR EL TAMAÑO DEL FORMULARIO WEB CON EL RATON

### DISCUSSION THREAD

**Discussion by Solarix57** [community/community#110148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

¿Como hago para ajustar el tamaño del formulario web en la vista de diseño y no me bloquee al hacerlo manualmente con el puntero en Visual Studio 2022?
Me sale un espacio en blanco delgado como si fuera el encabezado y de resto la ventana me aparece sombreada en gris y el simbolo de bloque y no me permite ajustar la pantalla manualmente con el ratón, ¿Eso a que se debe o como lo soluciono?
![Vista Diseño Visual Studio 2022](https://github.com/community/community/assets/159983615/782524c9-4cba-4374-be10-068b80ed6c4c)

**Comment by Akash1134** [comment]
Hi @Solarix57 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
